FileUtil/FileUpload.py

```
@@ -0,0 +1,45 @@
+from fastapi import UploadFile, HTTPException,status
+
+from common.config import settings
+import uuid
+import os
+
+class FileUtil:
+    ALLOWED_EXTENSIONS = ["image/png","image/jpeg","image/jpg"]
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)